src/client/BuilderClient.ts

```diff
@@ -30,6 +30,20 @@ export class BuilderClient {
     }
   }
 
+  /**
+   * Returns AuthIdentity either from the constant variable or the method given in the constructor.
+   */
+  private getIdentity(): AuthIdentity {
+    return this.identity instanceof Function ? this.identity() : this.identity
+  }
+
+  /**
+   * Returns an address either from the constant variable or the method given in the constructor.
```

Comment:
What constant value? I think that these JSDocs could be a bit more explicit as reading it without taking a peek at the implementation can be confusing? 
WDYT?